src/models/pacman.c

```diff
@@ -81,18 +81,23 @@ void drawPacman(int count)
     }
 
     // Get new pacman position in grid
-    struct Position newPacmanGridPos = uiPosToGridPos(pacmanPosCopy);
+    struct Position newPacmanGridPos = getUiPosToGridPos(pacmanPosCopy);
 
     if (!arePositionEquals(pacmanGridPos, newPacmanGridPos))
     {
-
-        if (isObstacle(newPacmanGridPos))
+        // If pacman, just blit him at without updating his position
+        if (getMazeElementAt(newPacmanGridPos) == WALL)
```

Comment:
Oh !
Why don't you use the isObstacle function anymore ?

Reply:
ab1dee0abbd74b8fb2f3085af625f0b676b9bb79